packages/appeals-service-api/src/validators/isEmail.js

```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line no-useless-escape
+var emailAddressFormat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
```

Comment:
I think we were hoping to use the in built isEmail function from express-validator to validate emails going forward rather than managing our own regexes